crates/vim/src/helix.rs

```diff
@@ -345,6 +345,9 @@ impl Vim {
                     cx,
                 );
             }
+
+            // Drop back to normal mode after yanking
+            vim.mode = Mode::HelixNormal;
```

Comment:
my only comment is that we might need to use `self.switch_mode(Mode::HelixNormal, true, window, cx)` here for consistency.

Reply:
Good catch, thanks!